changelog.dd

```diff
@@ -42,6 +43,12 @@ Macros:
     COREMODREF = <a href="phobos/core_$1.html">$2</a>
     XREF = <a href="phobos/std_$1.html#$2">$2</a>
     CXREF = <a href="phobos/core_$1.html#$2">$2</a>
+    OXREF = <a href="phobos/object.html#$2">$2</a>
+    NXREF = <a href="phobos/std_$1.html#.$2">$2</a>
```

Comment:
I just keep them in sync w/ what's available in dlang.org, so people don't fall back to plain html.